config/filesystems.php

```diff
@@ -7,5 +7,9 @@
             'driver' => 'local',
             'root' => getcwd(),
         ],
+        'local-recipes' => [
+            'driver' => 'local',
+            'root' => $_SERVER['HOME'] . '/.mise/Recipes',
```

Comment:
Good call! Could you refactor our other calls to this location to use this config too? Even if they don't use this filesystem driver, although that would be nice, they could at least use this config item.

Reply:
(Not necessarily a part of this PR, just could I ask you to do that?)